config/tomlConfig_test.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/multiversx/mx-chain-go/config"
+	"github.com/pelletier/go-toml"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadNodeOverrideConfigs(t *testing.T) {
+	t.Parallel()
+
+	testString := `
+# OverridableConfigTomlValues represents an array of items to be overloaded inside other configuration files, which can be helpful
```

Comment:
Test string can be shorter, to only have a short, dummy comment instead of the full description.

Reply:
made it shorter

Reply:
made it shorter here https://github.com/multiversx/mx-chain-simulator-go/pull/21